libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -48,7 +49,7 @@ bool ReferencesResolver::visit(Identifier const& _identifier)
 	auto declarations = m_resolver.nameFromCurrentScope(_identifier.name());
 	if (declarations.empty())
 	{
-		string const& suggestions = m_resolver.similarNameSuggestions(_identifier.name());
+		string const& suggestions = quotedAlternativesList(m_resolver.similarNameSuggestions(_identifier.name()));
```

Comment:
You cannot store this as a `const&`. I actually wonder why the compiler does not complain here...